scipy/stats/_resampling.py

```diff
@@ -207,11 +207,22 @@ def statistic(i, axis=-1, data=data_iv, unpaired_statistic=statistic):
     if not paired and n_samples > 1 and method == 'bca':
         raise ValueError(message)
 
+    message = "`bootstrap_result` must have attribute `bootstrap_distribution'"
+    if (bootstrap_result is not None
+            and not hasattr(bootstrap_result, "bootstrap_distribution")):
+        raise ValueError(message)
+
+    message = ("Either `bootstrap_result.bootstrap_distribution.size` or "
+               "`n_resamples` must be positive.")
+    if n_resamples_int == 0 and (not bootstrap_result or
+            not bootstrap_result.bootstrap_distribution.size):
```

Comment:
`scipy/stats/_resampling.py:218:13: E128 continuation line under-indented for visual indent`
```suggestion
    if ((not bootstrap_result or
         not bootstrap_result.bootstrap_distribution.size)
            and n_resamples_int == 0):
```
Other suggestions welcome.

Reply:
Fine with me if that passes the linter (never sure with `if` without running it)